src/Sunny.jl

```diff
@@ -101,9 +98,18 @@ include("SunnyGfx/SunnyGfx.jl")
 include("SunnyGfx/CrystalViewer.jl")
 export view_crystal, offline_viewers, browser
 
-include("ParallelTempering.jl")
+include("MonteCarlo/Samplers.jl")
```

Comment:
Can you put all the `include("MonteCarlo/...")` lines next to each other, following the style of previous imports?